### PR TITLE
Bar PR 2

### PR DESCRIPTION
**Pull request by muglug**

### CODE REVIEW

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 1]
```diff
@@ -2,6 +2,4 @@
 
 $a = 6;
 
-
-strlen($a);
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 1 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 2]
```diff
@@ -2,6 +2,4 @@
 
 $a = 6;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 2 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 3]
```diff
@@ -2,6 +2,4 @@
...
 
-
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 4]
```diff
@@ -2,6 +2,4 @@
 
 $a = 6;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 5]
```diff
@@ -2,6 +2,4 @@
...
 
-
-strlen($a);
-
+echo $b;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php [hunk 6]
```diff
@@ -3,5 +3,4 @@
 $a = 6;
 
 
-strlen($a);
-
+echo $b;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has tho

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — COMMENTED** [review]
fdsf

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

File: .travis.yml [hunk 7]
```diff
@@ -23,7 +23,6 @@ before_install:
     # disable default memory limit
     - echo memory_limit = 2G >> $INI
     - if [[ "$XDEBUG" = 'false' && -f $INI_DIR/xdebug.ini ]]; then phpenv config-rm xdebug.ini; fi
```
Comment (muglug) [inline .travis.yml:25]: fdsfss

File: .travis.yml [hunk 8]
```diff
@@ -23,7 +23,6 @@ before_install:
...
-    - composer clear-cache
```
Comment (muglug) [inline .travis.yml:26]: fdsfd

**Review by muglug — COMMENTED** [review]
blah

**Comment by muglug** [comment]
test

File: src/foo.php (hunk 6 above)
Comment (psalm-shepherd) [inline src/foo.php:6]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm has thoughts

File: src/foo.php (hunk 6 above)
Comment (psalm-shepherd) [inline src/foo.php:6]: Cannot find referenced variable $b in global scope

**Pull request closed without merging**